includes/Traits/Assets.php

```diff
@@ -39,13 +39,20 @@ trait Assets {
 	 *
 	 * @param string $handle Script handle.
 	 *
-	 * @return array
+	 * @return array Array containing combined contents of "<$handle>.asset.php" and "<$handle>.chunks.php".
 	 */
 	protected function get_asset_metadata( $handle ) {
-		$asset_file            = WEBSTORIES_PLUGIN_DIR_PATH . 'assets/js/' . $handle . '.asset.php';
-		$asset                 = is_readable( $asset_file ) ? require $asset_file : [];
+		$base_path = WEBSTORIES_PLUGIN_DIR_PATH . 'assets/js/' . $handle;
+
+		$asset_file  = $base_path . '.asset.php';
+		$chunks_file = $base_path . '.chunks.php';
```

Comment:
```suggestion
		$base_path = WEBSTORIES_PLUGIN_DIR_PATH . 'assets/js/';

		$asset_file  = $base_path . $handle . '.asset.php';
		$chunks_file = $base_path . $handle . '.chunks.php';
```

So that `$base_path` really is a _path_. Adds some redundancy, but makes it easier to comprehend.